Sources/PointFree/Admin/EmailPreview.swift

```diff
@@ -42,14 +42,13 @@ private func emailPreviewView(selectedTemplate: EmailTemplate?) -> Node {
         .fragment(options(selectedTemplate: selectedTemplate))
       )
     ),
-    .div(
+    .iframe(
```

Comment:
Better to render email previews in its own iframe so it's fully quarantined from the styles on this page.